CAM/SetGameplayCamRelativePitch.md

```diff
@@ -5,17 +5,17 @@ ns: CAM
 
 ```c
 // 0x6D0858B8EDFD2B7D 0x6381B963
-void SET_GAMEPLAY_CAM_RELATIVE_PITCH(float x, float Value2);
+void SET_GAMEPLAY_CAM_RELATIVE_PITCH(float x, float scalingFactor);
 ```
 
 ```
 Sets the camera pitch.  
 Parameters:  
 x = pitches the camera on the x axis.  
```

Comment:
```suggestion
Parameters:  
x = pitches the camera on the x axis.  
```
Moved to parameters section

Reply:
resolving as github sees this as identical

Reply:
updated the suggestion